views/prisonerProfile/adjudicationHistory.njk

```diff
@@ -119,4 +119,10 @@
         {{ mojPagination(pagination) }}
     {% endif %}
 
+		<div>
```

Comment:
BNTW we try to avoid using tabs on any of our files - the trouble is our code viewers are inconsistent when it comes to rendering them